### PR TITLE
[Catalog Promotion] Moved labels to separate twig

### PR DESCRIPTION
**Pull request by AdamKasp**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no/
| Related tickets | #13386 - this doc will be modified after this merge
| License         | MIT

to make customization easier we want to move catalog promotion labels to separate file

### CODE REVIEW

File: src/Sylius/Bundle/ShopBundle/Resources/views/Product/Box/_content.html.twig [hunk 1]
```diff
@@ -15,13 +15,11 @@
         <a href="{{ path('sylius_shop_product_show', {'slug': product.slug, '_locale': product.translation.locale}) }}" class="header sylius-product-name" {{ sylius_test_html_attribute('product-name', product.name) }}>{{ product.name }}</a>
 
         {% if not product.enabledVariants.empty() %}
-            {% set appliedPromotions = (product|sylius_resolve_variant).getAppliedPromotionsForChannel(sylius.channel) %}
             {% set price = money.calculatePrice(product|sylius_resolve_variant) %}
             {% set originalPrice = money.calculateOriginalPrice(product|sylius_resolve_variant) %}
+            {% set appliedPromotions = (product|sylius_resolve_variant).getChannelPricingForChannel(sylius.channel).getAppliedPromotions() %}
```
Comment (GSadee) [inline src/Sylius/Bundle/ShopBundle/Resources/views/Product/Box/_content.html.twig:20]: Why did you change the order? And I guess that `getAppliedPromotionsForChannel` method still exists

**Review by SirDomin — APPROVED** [review]

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thanks, Adam! :tada:
